### PR TITLE
Translate bot to English (New Strings)

### PR DESCRIPTION
**Pull request by robotenique**
Grande Robô ED #36

### CODE REVIEW

File: locales/en.po [hunk 1]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #: plugins_calculadora.js:23
 msgid "não foi definido nenhum numero para:"
-msgstr ""
+msgstr "no number has been defined for:"
 
 #: plugins_calculadora.js:27
 msgid "*Error ao Calcular:* `{{0}}`"
-msgstr ""
+msgstr "* Error when calculating: * `{{0}}`"
```
Comment (TiagoDanin) [inline locales/en.po]: > \*Error when calculating:\* \`{{0}}\`

Remove the space

File: locales/en.po [hunk 2]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #: plugins_calculadora.js:28
 msgid "Error ao Calcular: {{0}}"
-msgstr ""
+msgstr "Error Calculating: {{0}}"
 
 #: plugins_calculadora.js:33
 msgid "*Pelos meus calculos:* `{{0}} = {{1}}`"
-msgstr ""
+msgstr "* By my calculations: * `{{0}} = {{1}}`"
```
Comment (TiagoDanin) [inline locales/en.po]: > \*By my calculations:\* \`{{0}} = {{1}}\`

Same here ^

File: locales/en.po [hunk 3]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #: plugins_calculadora.js:34
 msgid "Resultado: {{0}}"
-msgstr ""
+msgstr "Result: {{0}}"
 
 #: plugins_callback.js:4
 msgid "Você já selecionou esta opção ‼️"
-msgstr ""
+msgstr "You already selected this option! ️"
```
Comment (TiagoDanin) [inline locales/en.po]: - String end without space

File: locales/en.po [hunk 4]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 #: plugins_commits.js:33
 msgid "Ver um commit."
-msgstr ""
+msgstr "View a commit."
 
 #. 0 = (number)
 #: plugins_dado.js:10
 msgid "⚠️ *Atenção:* O número mínimo de lados é `{{0}}`."
-msgstr ""
+msgstr "⚠️ * Attention: * The minimum number of sides is `{{0}}`."
```
Comment (TiagoDanin) [inline locales/en.po]: > ⚠️ \*Attention:\* The minimum number of sides is \`{{0}}\`.

File: locales/en.po [hunk 5]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #. 0 = (number)
 #: plugins_dado.js:14
 msgid "⚠️ *Atenção:* O alcance máximo de lados é `{{0}}`."
-msgstr ""
+msgstr "⚠️ * Warning: * The maximum range of sides is `{{0}}`."
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 6]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #. 0 = (number)
 #: plugins_dado.js:18
 msgid "⚠️ *Atenção:* O alcance máximo de dados é `{{0}}`."
-msgstr ""
+msgstr "⚠️ * Attention: * The maximum range of the die is `{{0}}`."
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 7]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
-#, fuzzy
 msgid "dados"
-msgstr "Dice"
+msgstr "die"
 
 #. 0 = (number), 1 = (Dados || Dado)
 #: plugins_dado.js:22
 msgid "*Rodando* `{{0}}` *{{1}}, "
-msgstr ""
+msgstr "* By running * `{{0}}` * {{1}},"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 8]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 #: plugins_google.js:8
 msgid "Não encontrei nada!"
-msgstr ""
+msgstr "I did not find anything!"
 
 #. No Translate <code> or </code>
 #: plugins_google.js:10
 msgid "<code>Sem resultados!</code>"
-msgstr ""
+msgstr "<code> No results! </code>"
```
Comment (TiagoDanin) [inline locales/en.po]: > \<code\>No results!\</code\>

File: locales/en.po [hunk 9]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #. 0 = (Search Text)
 #: plugins_google.js:26
 msgid "👁‍🗨 <b>Resultados para:</b> <code>{{0}}</code>\n"
-msgstr ""
+""
+msgstr "👁 <b> Results for: </b> <code> {{0}} </code>\n"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 10]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #: plugins_help.js:52
 msgid "_Eu enviei a mensagem de ajuda em uma mensagem privada._"
-msgstr ""
+msgstr "_I sent the help message in a private message._"
 
 #: plugins_help.js:71
 msgid "<b>Sobre:</b>"
-msgstr ""
+msgstr "<b> About: </b>"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 11]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #: plugins_help.js:74
 msgid "<b>Exemplo de uso:</b>"
-msgstr ""
+msgstr "<b> Example usage: </b>"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 12]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #: plugins_help.js:83
 msgid "<b>Categoria(s):</b>"
-msgstr ""
+msgstr "<b> Category (s): </b>"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 13]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #: plugins_help.js:85
 msgid "<b>Compatilibidade:</b>"
-msgstr ""
+msgstr "<b> Compatibility: </b>"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 14]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #: plugins_help.js:90
 msgid "Modo Inline"
-msgstr ""
+msgstr "Inline Mode"
 
 #: plugins_help.js:100
 msgid "<b>Escolha um plugin!</b>"
-msgstr ""
+msgstr "<b> Choose a plugin! </b>"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 15]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
-"informaçoes sobre o mesmo."
-msgstr ""
+msgid "Por favor, informe um dominio válido para que eu possa está enviando as informaçoes sobre o mesmo."
+msgstr "Please enter a valid domain so I can send you the information about it."
 
 #: plugins_ip.js:25
 msgid "*Hospedado por:* {{0}}\n"
-msgstr ""
+""
+msgstr "* Hosted by: * {{0}}\n"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 16]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
+""
 
 #: plugins_ip.js:28
 msgid "*Provedor:* {{0}}\n"
-msgstr ""
+""
+msgstr "* Provider: * {{0}}\n"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 17]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
+""
 
 #: plugins_ip.js:31
 msgid "*Cidade:* {{0}}\n"
-msgstr ""
+""
+msgstr "* City: * {{0}}\n"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 18]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
+""
 
 #: plugins_ip.js:34
 msgid "*Região:* {{0}}\n"
-msgstr ""
+""
+msgstr "* Region: * {{0}}\n"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 19]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
+""
 
 #: plugins_ip.js:36
 msgid "*Pais:* {{0}} - {{1}}"
-msgstr ""
+msgstr "* Country: * {{0}} - {{1}}"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 20]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
-msgstr ""
+msgid "Nome sorteado foi: <code>{{0}}</code>"
+msgstr "Choosen name was: <code> {{0}} </code>"
 
 #: plugins_sorteio.js:18
-msgid ""
-"Nomes para sorteio: <code>{{0}}</code>\n"
+msgid "Nomes para sorteio: <code>{{0}}</code>\n"
+"{{1}}"
+msgstr "Names for drawing: <code> {{0}} </code>\n"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 21]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
-msgstr ""
+msgstr "HTML Style"
 
 #. No remove '<b>' or '</b>'
 #: plugins_text.js:20
-msgid ""
-"<b>Texto estilo: HTML</b>\n"
+msgid "<b>Texto estilo: HTML</b>\n"
+"{{0}}"
+msgstr "<b> Text style: HTML </b>\n"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 22]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
+msgstr "Code Block"
 
 #. No remove '<b>', '</b>', <code> and </code>
 #: plugins_text.js:30
-msgid ""
-"<b>Texto estilo: Bloco de Código</b>\n"
+msgid "<b>Texto estilo: Bloco de Código</b>\n"
 "<code>{{0}}</code>"
-msgstr ""
+msgstr "<b> Text style: Code block </b>\n"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 23]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
+"<code> {{0}} </code>"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 24]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #: plugins_torrent.js:38
 msgid "Não há resultados"
-msgstr ""
+msgstr "No results"
 
 #: index.js:76
 msgid "*Ocorreu um erro* ao executar este comando."
-msgstr ""
+msgstr "* An error occurred* while executing this command."
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 25]
```diff
@@ -13,453 +13,454 @@ msgstr ""
...
 
 #: index.js:77
-msgid ""
-"Se o problema persistir contante a equipe de suporte via \\`/suporte sua "
-"mensagem.\\`"
-msgstr ""
+msgid "Se o problema persistir contate a equipe de suporte via \\`/suporte sua mensagem.\\`"
+msgstr "If the problem persists contact the support team via \\ `/support your message. \\`"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

File: locales/en.po [hunk 26]
```diff
@@ -488,17 +489,17 @@ msgstr "Tools"
...
 #: InfoAllPlugin.list:13
 msgid "Ajuda em funções de outros plugins"
-msgstr "Help on functions of others plugins"
+msgstr "Help with other plugins functions"
 
 #. Command example of Plugin (ID: callback)')
 #: InfoAllPlugin.list:15
 msgid "callback:same"
-msgstr "callback:same"
+msgstr "callback: same"
```
Comment (TiagoDanin) [inline locales/en.po]: > callback:same

File: locales/en.po [hunk 27]
```diff
@@ -758,36 +752,33 @@ msgstr "/ping"
...
 #: InfoAllPlugin.list:165
 msgid "Diversos estilos de formatação de textos."
-msgstr "Some funnies styles of text formatting. "
+msgstr "Several styles of text formatting."
 
 #. Command example of Plugin (ID: textformat)')
 #: InfoAllPlugin.list:167
 msgid "/texto *LINDO*"
-msgstr "/text *BEAUTIFUL*"
+msgstr "/ text *CUTE*"
```
Comment (TiagoDanin) [inline locales/en.po]: ^

**Comment by robotenique** [comment]
I changed some things, check now

**Comment by TiagoDanin** [comment]
Thanks

**Pull request merged**